<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0" colspan="3">
     <widget class="main_closewidget" name="close_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>801</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="main_topwidget" name="top_widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>801</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="mainwindows_rightwidget" name="right_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>541</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="myfilewg" name="myfiles_page"/>
      <widget class="sharelist" name="sharefile_page"/>
      <widget class="rankinglist" name="ranking_page"/>
      <widget class="transfer" name="transfer_page"/>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>544</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>741</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mainwindows_rightwidget</class>
   <extends>QWidget</extends>
   <header location="global">mainwindows_rightwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>main_topwidget</class>
   <extends>QWidget</extends>
   <header location="global">main_topwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>main_closewidget</class>
   <extends>QWidget</extends>
   <header location="global">main_closewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>myfilewg</class>
   <extends>QWidget</extends>
   <header location="global">myfilewg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>sharelist</class>
   <extends>QWidget</extends>
   <header location="global">sharelist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>rankinglist</class>
   <extends>QWidget</extends>
   <header location="global">rankinglist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>transfer</class>
   <extends>QWidget</extends>
   <header location="global">transfer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
